src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount = 0;
+  const price = 40;
+  const discountDaysThree = 20;
+  const discountDaysSeven = 50;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) the 5th point